docs/date.md

```diff
@@ -496,3 +496,17 @@ date = Temporal.Date.from('2006-08-24');
 Object.assign({}, date).day  // => undefined
 Object.assign({}, date.getFields()).day  // => 24
 ```
+
+### date.**getISOCalendarFields()**(): { year: number, month: number, day: number }
+
+**Returns:** a plain object with properties expressing `date` in the ISO 8601 calendar.
+
+This method is mainly useful if you are implementing a custom calendar.
+Most code will not need to use it.
+Use `date.getFields()` instead.
```

Comment:
Add a TODO here for giving an example of the "proper" way to get at the ISO fields.  We should fill it in once we decide on a few of the lingering API questions.

Reply:
I'm guessing by "proper" way you mean `date.withCalendar('iso8601').getFields()`? I've added this example to the calendar branch, so it will make it into the docs from there.